File: benchmark/fib.php
<?php
function fib($n) {
  if ($n < 2)
    return $n;
  return fib($n - 1) + fib($n - 2);
}

$n = $argv[1];
$m = $argv[2];
for ($i = 0; $i < $n; $i++)
  echo fib($m) . "\n";

File: benchmark/fib.py
import sys

def fib(n):
  if n < 2: return n
  return fib(n - 1) + fib(n - 2)

n = int(sys.argv[1])
m = int(sys.argv[2])
for i in range(0, n):
  print(fib(m))
